src/Symfony/Component/Security/Core/User/User.php

```diff
@@ -78,6 +78,16 @@ public function getSalt(): ?string
      * {@inheritdoc}
      */
     public function getUsername(): string
+    {
+        trigger_deprecation('symfony/security-core', '5.3', 'Method "%s" is deprecated and will be removed in 6.0, use getUserIdentifier() instead.', __METHOD__);
+
+        return $this->username;
+    }
+
+    /**
+     * @see getUsername
```

Comment:
I'm not sure we should point to a deprecated method, better duplicate the docs I think